tests/playback.js

```diff
@@ -117,6 +129,16 @@ queue_next_packet = function () {
 var bytes_processed = 0;
 
 do_packet = function () {
+    // Avoid having an excessive queue buildup
+    if (rfb._flushing) {
```

Comment:
unless I'm misreading this, I don't think we should be doing anything special in our performance testing code.  If the normal behavior is insufficient, the performance code should exhibit the same behavior, not work around it.